website/.lintstagedrc.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
```

Comment:
Maybe you can use [husky](https://www.npmjs.com/package/husky) instead of this config. You can check out the monorepo one.